src/braket/experimental/autoqasm/api.py

```diff
@@ -87,6 +90,30 @@ def gate(*args) -> Callable[[Any], None]:
     return _function_wrapper(args, _convert_gate)
 
 
+def pulse_sequence(*args, **kwargs):
+    if "implements" in kwargs:
+        assert len(args) == 0, "Cannot specify both `implements` and positional arguments."
```

Comment:
Why not? What if someone wants to do this inline instead of as a decorator, like:  
`aq.pulse_sequence(my_func, implements=rx)`